dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/beans/factory/annotation/ReferenceAnnotationBeanPostProcessor.java

```diff
@@ -60,7 +65,7 @@
  */
 public class ReferenceAnnotationBeanPostProcessor extends InstantiationAwareBeanPostProcessorAdapter
         implements MergedBeanDefinitionPostProcessor, PriorityOrdered, ApplicationContextAware, BeanClassLoaderAware,
-        DisposableBean {
+        DisposableBean, ApplicationListener<ContextRefreshedEvent> {
```

Comment:
If based on `ApplicationListener<ContextRefreshedEvent>` , the `@Reference` fields will be null on `@PostConsturct` or other life-cycle methos

Reply:
I always think that @EnableDubbo (basepackage="com.xxx") since it supports packet scanning, should avoid having the user repeatedly configure the same @ComponentScan(basepackage="com.xxx").